docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Quick Start - Arrays and Collections - C# Guide
+description: Learn C# in your browser
+keywords: C#, Get Started, tutorial, collections, List
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/13/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# C# Quick start: Arrays and Collections #
+
+This tutorial provides an introduction of the C# language and the basics of the <xref:System.Collections.Generic.List%601>
+class.
+
+## A simple list example.
+
+Create a directory named **list-quickstart** and run `dotnet new console`.
+
+Open **program.cs** in your favorite editor, and replace the existing code with the following:
+
+```csharp
+using System;
+using System.Collections.Generic;
+
+namespace list_quickstart
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var names = new List<string> { "<name>", "Ana", "Felipe" };
+            foreach (var name in names)
+            {
+            Console.WriteLine($"Hello {name.ToUpper()}!");
+            }
+        }
+    }
+}
+```
+
+> [!NOTE]
+> If you are starting from the code you wrote in [dot.net](https://dot.net/) you will already have that code written.
```

Comment:
Any chance this could go just under the heading, so users know immediately they can skip it?

*Not critical for this change:* I'm not sure if you can do dynamic content in docs, but I can easily pass a URL parameter if there is a way to change what is displayed?

Reply:
first one: done.

second: I don't know. I'll start a thread with @eisenbergeffect

Reply:
Don't worry too much about the second one, it's a minor optimization in the scheme of things 😄.